# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/ipc/RemoteAccessibleShared.h
# Commit: 86da8f2a28ea
# Full Hash: 86da8f2a28ea1992671d08dfb56c881a1a82daa5
# Author: James Teh <jteh@mozilla.com>
# Date: 2022-08-04 04:34:13
# Regressor Bug: 1781536
# File Overlap Count: 1
# Description:
#   Bug 1781536 part 3: Move RelationByType to base Accessible. r=morgan
#   
#   This required changing the RemoteAccessible implementations to return Relation instead of an array of RemoteAccessible.
#   Platform implementations have been updated to use the unified method where appropriate.
#   
# ==============================================================================

diff -r 95ae96802dbe -r 86da8f2a28ea accessible/ipc/RemoteAccessibleShared.h
--- a/accessible/ipc/RemoteAccessibleShared.h	Wed Aug 03 23:58:51 2022 +0000
+++ b/accessible/ipc/RemoteAccessibleShared.h	Wed Aug 03 23:58:51 2022 +0000
@@ -48,11 +48,7 @@
  */
 virtual already_AddRefed<AccAttributes> Attributes() override;
 
-/**
- * Return set of targets of given relation type.
- */
-virtual nsTArray<RemoteAccessible*> RelationByType(
-    RelationType aType) const override;
+virtual Relation RelationByType(RelationType aType) const override;
 
 bool IsSearchbox() const;
 